twingly-search-api-ruby.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "faraday",  [">= 0.9.2", "< 1.0"]
+  spec.add_dependency "faraday",  [">= 0.9.2", "<= 1.3.0"]
```

I think we could even allow something like this:

```suggestion
  spec.add_dependency "faraday",  [">= 0.9.2", "< 2.0"]
```

Otherwise we will have to change things here soon enough again. I would think Faraday tries to stick to SemVer.